Userland/Applications/PixelPaint/Tools/GradientTool.cpp

```diff
@@ -78,6 +80,11 @@ void GradientTool::on_mousemove(Layer* layer, MouseEvent& event)
         set_hover_flag(m_hover_over_start_handle, m_gradient_start.value());
         set_hover_flag(m_hover_over_drag_handle, m_gradient_center.value());
         set_hover_flag(m_hover_over_end_handle, m_gradient_end.value());
+
+        if (m_mode == GradientMode::Radial) {
+            set_hover_flag(m_hover_over_left_handle, m_gradient_left.value());
```

Comment:
`left` and `right` are not the best names; i think that `transversal_start` and `transversal_end` could be better… 🤔